bin/node/runtime/src/lib.rs

```diff
@@ -684,8 +724,8 @@ impl_runtime_apis! {
 				}
 			}
 
-			add_benchmark!(params, batches, b"bridge-eth-poa", BridgeEthPoA);
-			add_benchmark!(params, batches, b"bridge-currency-exchange", BridgeCurrencyExchangeBench::<Runtime>);
+			add_benchmark!(params, batches, b"bridge-eth-poa", BridgeKovan);
+			add_benchmark!(params, batches, b"bridge-currency-exchange", BridgeCurrencyExchangeBench::<Runtime, KovanCurrencyExchange>);
```

Comment:
shall we have benchmarks for Rialto as well?

Reply:
I thought about it, but given that we previously only benchmarked Kovan I wanted to keep it like that for now. We can look into adding Rialto once we update the benches to use the automated approach